python/pfs/instmodel/splinedPsf.py

```diff
@@ -18,11 +18,13 @@
 import scipy.ndimage.interpolation
 
 from . import spotgames
-from .spectrum import LineSpectrum, ArcSpectrum
+from .slit import Slit
 
 from . import psf
 reload(psf)
 
+NUM_FIBERS = 651  # Number of fibers per spectrograph
```

Comment:
Only used once in this file. Shouldn't this constant be used also for the fiber numbering in ``slit.py``?

Reply:
Sorry, neglected to address this before merging. I'll fix it up.